user_management.c

```diff
@@ -0,0 +1,129 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct {
+    int id;
+    char name[50];
+    int age;
+} User;
+
+void create_user() {
+    FILE* file = fopen("users.txt", "a");
```

Comment:
This is repeated in each and every function you can move it to one function so that this not repeated again and again